src/config/theme/lyrics.rs

```diff
@@ -0,0 +1,17 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Default, Clone)]
+pub struct LyricsConfig {
+    pub timestamp: bool,
+}
+
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Default)]
+pub struct LyricsConfigFile {
+    pub(super) timestamp: bool,
```

Comment:
Sorry I just noticed one more thing, can you add the default here?

```suggestion
    #[serde(default)]
    pub(super) timestamp: bool,
```